packages/terra-clinical-result/src/ClinicalResult.module.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .decorated-result-display {
+    align-items: baseline;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+
+    &.status-in-error del {
+      color: var(--terra-clinical-result-display-status-entered-in-error-color, rgba(28, 31, 33, 0.65));
+      text-decoration: line-through;
+    }
+
+    &.truncated {
+      flex-wrap: nowrap;
+
+      .result-display {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        word-break: normal;
+        word-wrap: normal;
+      }
+    }
+
+    .result-display {
+      overflow: visible;
+    }
+
+    .icon-modified {
+      color: var(--terra-clinical-result-display-icon-modified-color, #26a2e5);
+      flex: 0 0 auto;
+      height: var(--terra-clinical-result-display-icon-modified-height, 0.875em); // Must use ems for font scaling
+      margin-left: var(--terra-clinical-result-display-icon-modified-margin-left, 0.125em); // Must use ems for font scaling
+      overflow: hidden;
+      position: relative;
+      top: var(--terra-clinical-result-display-icon-modified-top); // Must use ems for font scaling
+      vertical-align: baseline;
+    }
+
+    .icon-comment {
+      color: var(--terra-clinical-result-display-icon-comment-color, rgba(28, 31, 33, 0.65));
+      flex: 0 0 auto;
+      height: var(--terra-clinical-result-display-icon-comment-height, 1em); // 0.9375em; Must use ems for font scaling
+      margin-left: var(--terra-clinical-result-display-icon-comment-margin-left, 0.125em); // Must use ems for font scaling
+      overflow: hidden;
+      position: relative;
+      top: var(--terra-clinical-result-display-icon-comment-top, 0.125em); // Must use ems for font scaling
+      vertical-align: baseline;
+    }
+
+    .icon-unverified {
+      flex: 0 0 auto;
+      height: var(--terra-clinical-result-display-icon-unverified-height, 0.85em); // Must use ems for font scaling
```

Comment:
Is this missing height & color?

Reply:
Color - no - this one has an included color and already themeable as a controlled icon.

do you mean something other than height?